sdk/validator/validator.go

```diff
@@ -70,12 +69,7 @@ func GetValidatorInfo(net network.Network, validationID ids.ID) (platformvm.L1Va
 	return vdrInfo, nil
 }
 
-func GetValidationID(rpcURL string, nodeID ids.NodeID, validatorManagerAddressStr string) (ids.ID, error) {
-	managerAddress := common.HexToAddress(validatorManagerAddressStr)
-	return GetRegisteredValidator(rpcURL, managerAddress, nodeID)
-}
-
-func GetRegisteredValidator(
+func GetValidationID(
```

Comment:
Can we add description to this function?

// Verifies
 /**
     * @notice Returns a validation ID registered to the given nodeID
     * @param nodeID ID of the node associated with the validation ID
     */